src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig

```diff
@@ -0,0 +1,126 @@
+        "label" (string) (default: '') Label of link
+        "path" (string) (default: '') Url of link
+      },
+      ...
+    ],
+    - "navigation_text" (string) (default: ''): Text of navigation in breadcrumb
+    - "ellipsis_label" (string) (default: ''): Label of the ellipsis, e.g. "Click to expand"
+    - "icon_file_path" (string) (default: ''): URL to icons file
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (optional) (string)
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _breadcrumb_size = links|length %}
+{% set _navigation_text = navigation_text|default('') %}
+{% set _ellipsis_label = ellipsis_label|default('') %}
+{% set _ellipsis_text = ellipsis_text|default('…') %}
+{% set _breadcrumb_segment_class = 'ecl-breadcrumb-core__segment' %}
+{% set _min_items_left = min_items_left|default(1) %}
+{% set _min_items_right = min_items_right|default(2) %}
+{% set _nav_attributes = [
+  { name: 'aria-label', value: _navigation_text },
+  { name: 'data-ecl-breadcrumb-core', value: 'true' }]
+%}
+{% set _breadcrumb_icon = {
+  icon: {
+    type: 'ui',
+    name: 'corner-arrow',
+    size: '2xs',
+    transform: 'rotate-90',
+    path: icon_file_path|default(''),
+  },
+  extra_classes: 'ecl-breadcrumb-core__icon',
+  extra_attributes: [{ name: 'role', value: 'presentation' }]
+} %}
+
+{% set _css_class = 'ecl-breadcrumb-core' %}
+
+{# Internal logic - Process properties #}
+
+{% if _breadcrumb_size > (_min_items_left + _min_items_right) %}
+  {% set _nav_attributes = _nav_attributes|merge([{ name: 'data-ecl-auto-init', value: 'BreadcrumbCore' }]) %}
+{% endif %}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% set extra_attributes = extra_attributes|merge(_nav_attributes) %}
+{% else %}
+  {% set extra_attributes = _nav_attributes %}
+{% endif %}
+
+{% for attr in extra_attributes %}
+  {% if attr.value is defined %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value|e('html_attr') ~ '"' %}
+  {% else %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name %}
+  {% endif %}
+{% endfor %}
+
+{# Print the result #}
+
+<nav class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  <ol class="ecl-breadcrumb-core__container">
+    {% for key, link in links %}
+      {% set _breadcrumb_segment_attributes = 'data-ecl-breadcrumb-core-item="static"' %}
+      {# Last item is not a link. #}
+      {% if loop.last %}
+        {% set _breadcrumb_segment_attributes = _breadcrumb_segment_attributes ~ ' aria-current="page"' %}
+        <li class="{{ _breadcrumb_segment_class ~ ' ecl-breadcrumb-core__current-page' }}" {{ _breadcrumb_segment_attributes|raw }}>
```

Comment:
You could use _css_class {{ _css_class ~ '__current-page '}}

Reply:
true but i find it very confusing, although we have sometimes used this approach, i would  avoid it and here i did that on purpose.